guides/hack/10-types/79-type-aliases.md

```diff
@@ -33,8 +38,9 @@ function distance(Point $p1, Point $p2): float {
 }
 ```
 
-Being in the same source file as the alias definition, the functions `create_Point` and `distance` have direct access to the integer
-fields in any `Point`'s tuple.  However, any file that includes this file does not.
+## `Type` v. `Newtype` | (Transparent v. Opaque)
```

Comment:
I think we should spell out 'versus' rather than writing 'v.', and I'm not sure what `|` means here. How about "Choosing between `type` and `newtype`"?

Also, we should only put valid syntax between backticks, and `type` and `newtype` are case sensitive.

Reply:
That's fair. Stylistically, I always write `v.` versus `vs.` but the whole word is probably better, and less confusing. Same for `|` as shorthand for or. Good feedback all around.